modules/logger/server/controllers/logger.js

```diff
@@ -110,6 +110,26 @@ function Logger() {
 
     return meta;
   }
+
+  /**
+   * endpoint controller for relaying client side log messages to server
+   * and possibly elasticsearch
+   * @param {String} req.body.level
+   * @param {Object} req.body.args
+   */
+  function relayLog(req, res, next) {
+    // level must be set. args may be anything but undefined
+    if (req.body.level && req.body.args !== undefined) {
+      logger.log(req.body.level, req.body.args);
+      res.status(200).send();
```

Comment:
Shouldn't this be a different status code for no content?  I can't remember exactly, but I thought there was one specific to a empty response 'OK'